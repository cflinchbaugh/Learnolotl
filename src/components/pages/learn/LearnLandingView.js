import React from 'react';
import styled from 'styled-components';

import Learn from './Learn';
import LearnDataSelect from './LearnDataSelect';


const StyleWrapper = styled.div`
    .demo-wrapper,
    .uploader-wrapper {
        display: inline-block;
        text-align: center;
    }
    .demo-wrapper {
        margin-bottom: 20px;
        margin-left: 0;
        margin-right: auto;
    }
    .uploader-wrapper {
        margin-left: auto;
        margin-right: 0;
    }

    @media (min-width: 1200px) {
        .demo-wrapper {
            margin-bottom: 0;
            border-bottom: none;
        }
    }
`

function LearnLandingView (props) {
    return (
        <StyleWrapper className="learn-container-wrapper mdCard col-xs-12">
            <div className="demo-wrapper
                col-xs-12
                col-lg-6
            ">
                <LearnDataSelect {...props.learnDataSelectData} />
            </div>
            <div className="uploader-wrapper
                col-xs-12
                col-lg-6
            ">
                <Learn {...props.learnData}/>
            </div>
        </StyleWrapper>
    );
}


export default LearnLandingView;