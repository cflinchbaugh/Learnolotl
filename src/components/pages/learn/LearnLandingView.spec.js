import React from 'react';
import {shallow} from 'enzyme';
import LearnLandingView from './LearnLandingView';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('LearnLandingView', () => {
    it('renders and matches snapshot', () => {
        const renderer = new ShallowRenderer();

        const learnLandingViewData = {
            
        }

        const tree = renderer.render(
            <LearnLandingView {...learnLandingViewData}/>
        );

        expect(tree).toMatchSnapshot();
    });

    
});