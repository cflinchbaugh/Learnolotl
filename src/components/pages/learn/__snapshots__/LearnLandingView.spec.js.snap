// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LearnLandingView renders and matches snapshot 1`] = `
<styled.div
  className="learn-container-wrapper mdCard col-xs-12"
>
  <div
    className="logo-wrapper col-xs-12"
  >
    <img
      src="test-file-stub"
    />
    <h1>
      Learnolotl
    </h1>
  </div>
  <div
    className="demo-wrapper
 col-xs-12
 col-lg-6
 "
  >
    <LearnDataSelect />
  </div>
  <div
    className="uploader-wrapper
 col-xs-12
 col-lg-6
 "
  >
    <Learn />
  </div>
</styled.div>
`;
